code/game/objects/empulse.dm

```diff
@@ -49,3 +49,47 @@ proc/empulse(turf/epicenter, heavy_range, light_range, log=0)
 			log_and_message_admins("EMPDEBUG: [T.name] - [T.type] - took [world.timeofday - time]ds to process emp_act()!")
 		#endif
 	return 1
+
+proc/empulse_exclusive(turf/epicenter, heavy_range, light_range, log=FALSE, list/exclude)

```

Comment:
This proc is (unsurprisingly) almost identical to empulse(). Why don't you just add list/exclude to empulse() instead of creating an entirely new proc?

Reply:
I suggested the same. So ye, do that, merge the functionality into `empulse` instead of creating another proc.

Reply:
That would require re-writing all other empulse calls everywhere.

Reply:
@skull132 Sorry, I thought you meant make separate proc